includes/sanitizers/class-amp-img-sanitizer.php

```diff
@@ -324,14 +325,15 @@ private function adjust_and_replace_node( $node ) {
 				&& 'figure' === $node->parentNode->nodeName
 				&& preg_match( '/(^|\s)(alignwide|alignfull)(\s|$)/', $node->parentNode->getAttribute( 'class' ) )
 			) {
-				$new_attributes['layout'] = 'responsive';
+				$new_attributes['layout'] = Layout::RESPONSIVE;
 			} else {
-				$new_attributes['layout'] = 'intrinsic';
+				$new_attributes['layout'] = Layout::INTRINSIC;
 			}
 		}
 
-		// Remove sizes attribute since it causes headaches in AMP and because AMP will generate it for us. See <https://github.com/ampproject/amphtml/issues/21371>.
-		unset( $new_attributes['sizes'] );
+		if ( isset( $new_attributes['sizes'] ) ) {
+			$new_attributes['disable-inline-width'] = '';
```

Comment:
You should use `Attribute` here.
```suggestion
		if ( isset( $new_attributes[ Attribute::SIZES ] ) ) {
			$new_attributes[ Attribute::DISABLE_INLINE_WIDTH'] = '';
```
Note: You'll have to add the `DISABLE_INLINE_WIDTH` constant to the `Layout` class as well.

Reply:
Constants everywhere :smile:!